docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -25,12 +25,12 @@ A component type that allows to define a container image-based configuration of
         - name: mavenrepo
           path: /root/.m2
        env:
-         - name: ENV_VAR
```

Comment:
Not all name fields have lowercase restrictions. The ones that have limits should be explicitly documented in the corresponding field. Therefore, there is no need to update this one.